client/services/services.go

```diff
@@ -3,10 +3,12 @@
 //go:generate mockgen -destination=./mocks/ad_groups.go -package=mocks . ADGroupsClient
 //go:generate mockgen -destination=./mocks/ad_service_principals.go -package=mocks . ADServicePrinicpals
 //go:generate mockgen -destination=./mocks/ad_users.go -package=mocks . ADUsersClient
+//go:generate mockgen -destination=./mocks/network.go -package=mocks . WatchersClient,VirtualNetworksClient
 //go:generate mockgen -destination=./mocks/monitor.go -package=mocks . LogProfilesClient
-//go:generate mockgen -destination=./mocks/network.go -package=mocks . VirtualNetworksClient,SecurityGroupsClient,WatchersClient
```

Comment:
SecurityGroupsClient dropped unintentionally?

Reply:
might be. thanks